src/test/resources/cucumber/pagination/StudentTest.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Check for Student MailId
+
+  Scenario: Retrieve mailId of a student
+    Given Student Data
+    When Key value "Email" is "Student11@gmail.com"
+    Then Result size should be 1
```

Comment:
Can we check the result as well instead of just size?

Reply:
As in store all expected rows and compare them individually against the result returned ?

Reply:
Yes what if the returned result has some other object or the one that's not expected?

Reply:
no way..email id is unique to every student

Reply:
I can understand that from a test case perspective we should have a look on what we have got in the result.
What if I have got some other object?

Reply:
As discussed please raise an issue and we can fix this later.